src/styles/styles.less

```diff
@@ -129,19 +130,7 @@ body {
 body.native {
     @media (prefers-color-scheme: dark) {
         .sb1ds-intro::after {
-            content: '';
-            display: block;
-            height: 335px;
```

Comment:
Denne linje gjør at bakken på fremsiden hopper opp om man slår på darkmode. Alle css regler her med unntak `background-image` er copy og paste fra den defaulte styling. Men `height` med 335px ble da feil fordi det finnes en annen media query som går på skjermstørrelse og endre height til 380px på desktop. Da blir disse 335px litt for kort.